content/applications/finance/accounting/payments.rst

```diff
@@ -30,114 +32,149 @@ At this point, the customer invoice or vendor bill is marked as :guilabel:`In pa
+   written off directly.
+
+.. _accounting/payments/batch-payments-matching:
 
 Batch payments matching
 -----------------------
 
-You can use the batch reconciliation feature to reconcile several outstanding payments or invoices
+Use the batch reconciliation feature to reconcile several outstanding payments or invoices
 simultaneously for a specific customer or vendor. Go to :menuselection:`Accounting --> Reporting -->
-Aged Receivable / Aged Payable`. You can see all transactions that have not been reconciled yet for
-that partner, and when you select a customer or vendor, the :guilabel:`Reconcile` option is
+Aged Receivable / Aged Payable`. All transactions that have not yet been reconciled for that
+partner are visible, and when a customer or vendor is selected, the :guilabel:`Reconcile` option is
 displayed.
 
 .. image:: payments/reconcile-option.png
    :alt: See the reconcile option.
 
+.. _accounting/payments/partial-payment:
+
 Registering a partial payment
 =============================
 
-To register a **partial payment**, click :guilabel:`Register payment` from the related invoice or
-bill, and enter the amount received or paid. Upon entering the amount, a prompt appears to decide
-whether to :guilabel:`Keep open` the invoice or bill, or :guilabel:`Mark as fully paid`. Select
-:guilabel:`Keep open` and click :guilabel:`Create payment`. The invoice or bill is now marked as
-:guilabel:`Partial`. Select :guilabel:`Mark as fully paid` if you wish to settle the invoice or bill
-with a difference in the amount.
+To register a partial payment, click on :guilabel:`Register Payment` from the related invoice or
+bill to open a new payment window.
+In the case of a partial payment (when the :guilabel:`Amount` paid is less than the total remaining
+amount on the invoice or the bill), the :guilabel:`Payment Difference` field displays the
+outstanding balance. There are two options:
+
+- :guilabel:`Keep open`: keep the invoice or the bill open and mark it with a :guilabel:`Partial`
+  banner;
+- :guilabel:`Mark as fully paid`: In this case, select an account in the
+  :guilabel:`Post Difference In` field and change the :guilabel:`Label` if needed. A journal entry
+  will be created to balance the accounts payable or receivable with the selected account.
+
+.. image:: payments/partial-payment.png
+   :alt: register a partial payment
 
-.. image:: payments/payment-difference.png
-   :alt: Partial payment of an invoice or bill.
+.. _accounting/payments/reconciling-payments:
 
 Reconciling payments with bank statements
 =========================================
 
 Once a payment has been registered, the status of the invoice or bill is :guilabel:`In payment`. The
 next step is :doc:`reconciling <bank/reconciliation>` it with the related bank statement line to
-have the transaction finalized and the invoice or bill marked as :guilabel:`Paid`.
+finalize the transaction and mark the invoice or bill as :guilabel:`Paid`.
```

Comment:
Let's also update "bank statement line" to "bank transaction" here, and then let's change "transaction" on the next line to make sure it isn't confusing. I suggest "to finalize the payment workflow" but I'm open to other suggestions if you prefer something different :)